Vwrs configuration section (#1199)
Macrometacorp/docs: docs/photoniq/vwrs/03-configuration-and-setup/03-configure-vwrs-waitingroom-page.md

---
sidebar_position: 30
title: Explore the VWRs Waiting Room Page
---

> The VWRs waiting room page is a customizable HTML page that users see when they are directed to the waiting room. The waiting room page is hosted on the Akamai NetStorage and configured using the `waitingRoomPath` option in the VWRs EdgeWorker `handleVwrsRequest()` function.

Reviewer: ```suggestion
The VWRs waiting room page is a customizable HTML page that users see when directed to the waiting room. It is hosted on Akamai's NetStorage and configured using the `waitingRoomPath` option in the VWRs EdgeWorker `handleVwrsRequest()` function.
```

  </body>
</html>
```

Within the above template, the class attributes `queue-position`, `queue-depth`, and `avg-wait-time` displays the visitor's current position in the queue, the total number of users in the waiting room, and the average waiting time, respectively.

## Configuring the Waiting Room Page

> The waiting room path must be configured as a [Conditional Origin](https://techdocs.akamai.com/property-mgr/docs/conditional-origins) in Akamai's property manager. This allows the waiting room HTML page to be served from the Akamai NetStorage. 

Reviewer: ```suggestion
The waiting room path must be configured as a [Conditional Origin](https://techdocs.akamai.com/property-mgr/docs/conditional-origins) in Akamai's property manager. This allows the waiting room HTML page to be served from the Akamai's NetStorage. 
```
